examples/computer_vision/efficientdet_pytorch/model_def.py

```diff
@@ -0,0 +1,409 @@
+
+        if self.args.prefetcher:
+            self.train_mean, self.train_std, self.train_random_erasing = self.calculate_means(mean=self.input_config['mean'], std=self.input_config['std'], re_prob=self.args.reprob, re_mode=self.args.remode, re_count=self.args.recount)
+            
+            self.val_mean, self.val_std, self.val_random_erasing = self.calculate_means(self.input_config['mean'], self.input_config['std'])
+
+        # self.val_reducer = self.context.experimental.wrap_reducer(self.validation_reducer, for_training=False)
+
+        
+    def build_callbacks(self) :
+        return {'model': self.model_ema.callbackobject()}
+
+    def calculate_means(self,
+            mean=IMAGENET_DEFAULT_MEAN,
+            std=IMAGENET_DEFAULT_STD,
+            re_prob=0.,
+            re_mode='pixel',
+            re_count=1):
+        # We need to precalculate the prefetcher. 
+        mean = torch.tensor([x * 255 for x in mean]).cuda().view(1, 3, 1, 1)
+        std = torch.tensor([x * 255 for x in std]).cuda().view(1, 3, 1, 1)
+        
+        if re_prob > 0.:
+            random_erasing = RandomErasing(probability=re_prob, mode=re_mode, max_count=re_count)
+        else:
+            random_erasing = None
+        
+        return mean, std, random_erasing
+
+    def _create_loader(self,
+            dataset,
+            input_size,
+            batch_size,
+            is_training=False,
+            use_prefetcher=True,
+            re_prob=0.,
+            re_mode='pixel',
+            re_count=1,
+            interpolation='bilinear',
+            fill_color='mean',
+            mean=IMAGENET_DEFAULT_MEAN,
+            std=IMAGENET_DEFAULT_STD,
+            num_workers=4,
+            distributed=False,
+            pin_mem=False,
+            anchor_labeler=None,
+        ):
+
+            print ('input_size', input_size)
+            print ('batch_size', batch_size )
+            print ('is_training', is_training)
+            print ('use_prefetcher', use_prefetcher)
+            print ('re_prob', re_prob)
+            print ('re_mode', re_mode)
+            print ('re_count', re_count)
+            print ('interpolation', interpolation)
+            print ('fill_color', fill_color)
+            print ('mean', mean)
+            print ('std', std)
+            print ('num_workers', num_workers)
+            print ('distributed', distributed)
+            print ('pin_mem', pin_mem)
+            print ('anchor_labeler', anchor_labeler)
+
+            if isinstance(input_size, tuple):
+                img_size = input_size[-2:]
+            else:
+                img_size = input_size
+
+            if is_training:
+                transform = transforms_coco_train(
+                    img_size,
+                    interpolation=interpolation,
+                    use_prefetcher=use_prefetcher,
+                    fill_color=fill_color,
+                    mean=mean,
+                    std=std)
```

Comment:
This is super nitpicky and totally a style thing, but can we move the last parentheses around a function call to the next line if each argument is a separate line? See the definition of `loader` below.